src/main/java/io/github/terra121/projection/GeographicProjection.java

```diff
@@ -277,4 +278,23 @@ public float azimuth(double x, double y, float angle) throws OutOfProjectionBoun
 	public enum Orientation {
 		none, upright, swapped
 	}
+	public static boolean cardinalDirectionCheck(CardinalDirection direction, float azimuth) {
```

Comment:
Sorry I'm bad at explaining things. This shouldn't be static, see the template I sent on Discord.